svc/TrackService.ts

```diff
@@ -78,34 +78,40 @@ export class TrackService extends HoistService {
     //------------------
     private async doTrackAsync(options: TrackOptions) {
         try {
-            const params: any = {
+            const query: any = {
                 msg: stripTags(options.message),
-                clientUsername: XH.getUsername()
+                clientUsername: XH.getUsername(),
+                appVersion: XH.getEnv('clientVersion'),
+                url: window.location.href
             };
 
-            if (options.category) params.category = options.category;
-            if (options.data) params.data = JSON.stringify(options.data);
-            if (options.severity) params.severity = options.severity;
-            if (options.logData !== undefined) params.logData = options.logData.toString();
-            if (options.elapsed !== undefined) params.elapsed = options.elapsed;
+            if (options.category) query.category = options.category;
+            if (options.data) query.data = JSON.stringify(options.data);
```

Comment:
the stringify here is funny -- normally in this format, would just be cleaner to submit it in object form as part of the overall object.  Would need to handle slightly different on the server.

Reply:
On the server it wouldn't require "processing" right? It would just work out of the box?

Reply:
Removed stringification here and added processing on server to accommodate